packages/nvidia_nat_ragas/src/nat/plugins/ragas/rag_evaluator/atif_evaluate.py

```
@@ -0,0 +1,138 @@
+        tqdm_position = TqdmPositionRegistry.claim()
+        first_metric_name = self.metrics[0].name
+        pbar = tqdm(total=len(ragas_dataset), desc=f"Evaluating Ragas {first_metric_name}", position=tqdm_position)
+        try:
+            results_dataset = ragas_evaluate(dataset=ragas_dataset,
+                                             metrics=self.metrics,
+                                             show_progress=True,
+                                             llm=self.evaluator_llm,
+                                             run_config=RunConfig(max_workers=self.max_concurrency),
+                                             _pbar=pbar)
+        except Exception as e:
+            logger.exception("Error evaluating ATIF ragas metric, Error: %s", e)
```

Ruff (0.15.4)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)